tests/ttnn/distributed/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+add_executable(test_distributed test_distributed.cpp)
+
+# Common function to set up target properties
+function(setup_ttnn_test_target target_name)
```

Comment:
Copy pasta?

Reply:
I've moved this up and commonized it in the parent CMakeLists. Thanks!